content/integrations/avascan.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 title: Avascan
 category: Block Explorers
-available: ["C-Chain"]
+available: ["C-Chain", "X-Chain", "P-Chain", "DFK", "Dexalot", "UPTN", "Lamina1", "Shrapnel", "XANA-CHAIN", "Playdapp", "Meld", "Beam", "PLAYA3ULL Games", "DOS", "Numbers", "StraitsX"]
```

Comment:
We want to avoid adding this many chains as it will add clutter to the search section.
For now please put 
```
available: ["C-Chain", "All EVM L1s"]
```

Reply:
but these are not EVM L1s, that's the list of avalanche subnets

Reply:
perfect, same for the other PR. Thanks

Reply:
@ashucoder9 ^

Reply:
up

Reply:
please change available tags. the options are "C-Chain", "All Avalanche L1s" or "All EVM L1s".

For this I would put `["C-Chain", "All Avalanche L1s"]